app/templates/macros/toolkit_forms.html

```diff
@@ -126,6 +126,24 @@
   {% endwith %}
 {%- endmacro %}
 
+{% macro checkbox_tree(question_content, service_data, errors, question_number=None, get_question=None) -%}
```

Comment:
The rest of the macros in this file have changed to use kwargs instead of named params.
So you can do
```
{% macro checkbox_tree(question_content, service_data, errors) -%}
...
     question=question_content.question|question_references(kwargs.get_question)
     hint=(question_content.hint or '')|question_references(kwargs.get_question),
```
etc.